ui/main/src/app/services/userActionLogs/UserActionLogsService.ts

```
@@ -0,0 +1,26 @@
+
+export class UserActionLogsService {
+    private static userActionLogsServer: UserActionLogsServer;
+
+    public static setUserActionLogsServer(userActionLogsServer: UserActionLogsServer): void {
+        UserActionLogsService.userActionLogsServer = userActionLogsServer;
+    }
+
+    public static queryUserActionLogs(filters: Map<string, string[]>): Observable<ServerResponse<Page<UserActionLog>>> {
+        return UserActionLogsService.userActionLogsServer.queryUserActionLogs(filters);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)